lambdas/delete-user-data/src/config.ts

```diff
@@ -1,3 +1,5 @@
 export const config = {
+  componentId: "ipv-core-delete-user-data",
```

Comment:
`componentId` should be the URI representing core. In production this is https://identity.account.gov.uk and is the config param at `core/self/componentId`

Reply:
Changed this to now get the componentId from SSM